textinput.go

```diff
@@ -61,10 +65,16 @@ func (p *textinputPlugin) InitPluginGLFW(window *glfw.Window) error {
 	p.channel.HandleFuncSync("TextInput.setClient", p.handleSetClient)
 	p.channel.HandleFuncSync("TextInput.clearClient", p.handleClearClient)
 	p.channel.HandleFuncSync("TextInput.setEditingState", p.handleSetEditingState)
-
+	// Desktop's don't have a virtual keyboard, so there is no need to show or hide it"
+	p.channel.HandleFuncSync("TextInput.show", p.handleIgnore)
 	return nil
 }
 
+// HandleIgnore is used to ignore method call
+func (p *textinputPlugin) handleIgnore(arguments interface{}) (reply interface{}, err error) {
```

Comment:
Make `handleIgnore` available for other plugin, (Only for `textinputPlugin` atm)
Move `handleIgnore` into method-channel.go